app/src/main/res/layout-sw600dp/administrator_controls_profile_view.xml

```diff
@@ -37,14 +37,14 @@
       android:id="@+id/edit_profiles_text_view"
       style="@style/Body"
       android:layout_width="match_parent"
-      android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex? @color/admin_highlighted : @color/white}"
+      android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex? @color/admin_highlighted : @color/admin_controls_options_background_color}"
```

Comment:
Can we change this color too: `admin_highlighted`? name it more correctly.

Reply:
Done.